src/main/scala/org/codeminers/standalone/ReplMain.scala

```diff
@@ -0,0 +1,34 @@
+package org.codeminers.standalone
+
+import io.joern.console.{BridgeBase, JoernProduct}
+import io.joern.joerncli.console.Predefined
+
+/** Extend/use joern as a REPL application */
+object ReplMain extends BridgeBase {
+
+  def main(args: Array[String]): Unit = {
+    run(parseConfig(args))
+  }
+
+  override protected def predefLines = {
+    Predefined.forInteractiveShell
+//      .map {
+//      case line if line.contains("docSearchPackages") =>
+////        val customDocSearchPackagesFullname = DocSearchPackages.getClass.getName.dropRight(1) // dropping `$` object name
+////        val customDocSearchPackagesFullname = getClass.getName.dropRight(1) // dropping `$` object name
+////        s"import _root_.$customDocSearchPackagesFullname"
+//        "TODO"
+//      case other => other
+//    }
+    ++ Seq(
+      s"import _root_.${getClass.getPackageName}.*"
```

Comment:
@mpollmeier Would custom language extensions be brought into this REPL?

Reply:
you mean the custom steps etc? yes they are available, including the `.help` entries...
i added a few examples to the readme

Reply:
Great!